packages/exporter-trace-otlp-http/test/node/CollectorTraceExporter.test.ts

```diff
@@ -354,4 +409,44 @@ describe('OTLPTraceExporter - node with json over http', () => {
       });
     });
   });
+  describe('export - with timeout', () => {
+    beforeEach(() => {
+      fakeRequest = new Stream.PassThrough();
+      stubRequest = sinon.stub(http, 'request').returns(fakeRequest as any);
+      spySetHeader = sinon.spy();
+      (fakeRequest as any).setHeader = spySetHeader;
+      collectorExporterConfig = {
+        headers: {
+          foo: 'bar',
+        },
+        hostname: 'foo',
+        attributes: {},
+        url: 'http://foo.bar.com',
+        keepAlive: true,
+        httpAgentOptions: { keepAliveMsecs: 2000 },
+        timeoutMillis: 100,
+      };
+      collectorExporter = new OTLPTraceExporter(collectorExporterConfig);
+      spans = [];
+      spans.push(Object.assign({}, mockedReadableSpan));
+    });
+    it('should log the timeout request error message', done => {
+      const responseSpy = sinon.spy();
+      collectorExporter.export(spans, responseSpy);
+
+      setTimeout(() => {
+        fakeRequest.emit('error', { code: 'ECONNRESET'});
```

Comment:
Why emit an error after 100ms which is exactly the request timeout? isn't this some sort of race?